db/models/donation.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Donation = db.define('donations', {
+  date: {
+    type: Sequelize.DATE,
+    defaultValue: Sequelize.NOW
+  },
+  amount: {
+    type: Sequelize.INTEGER, // in U.S. Cents
+    allowNull: false
+  },
+  method: {
+    type: Sequelize.ENUM('stripe', 'check', 'other')
+  },
+  thanked: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
+  dedication: {
+    type: Sequelize.STRING
```

Comment:
Oh I just meant - though we have a thanked flag - we wouldn't know in what manner they'd want to be thanked (whether over Twitter, or like a community Slack channel, or in a non-existent email newsletter to all of our adoring fans, etc.)